x/pairing/keeper/unstaking.go

```diff
@@ -130,15 +126,24 @@ func (k Keeper) creditUnstakingEntries(ctx sdk.Context, entriesToUnstake []epoch
 	}
 }
 
-func (k Keeper) unstakeHoldBlocks(ctx sdk.Context, chainID string) (uint64, error) {
-	spec, found := k.specKeeper.GetSpec(ctx, chainID)
+// TODO: this is duplicated in x/dualstaking/keeper/delegate.go; merge into one call
```

Comment:
do this TODO :)

Reply:
at a second glance this isn't so useful, because the first parts anyhow depends on the calling module; also, will need to add epochstorage as dependency to spec module, which is more trouble than this refactor would save. so instead, I changed to comment to note that any changes should be reflected in both sites. 

fixed 31633a97